external/include/mbedtls/config.h

```diff
@@ -2627,7 +2627,8 @@
 #undef MBEDTLS_ERROR_STRERROR_DUMMY
 #undef MBEDTLS_GENPRIME
 
-#undef MBEDTLS_FS_IO
+/* Enabled FS_IO for CURL */
+//#undef MBEDTLS_FS_IO
```

Comment:
Commented undefinition means enabled??
```#undef``` and nothing (commented) are same. Isn't it?

Reply:
In mBED TLS config, All the functionalities are enabled on top and few functionalities are disabled for light device. FS_IO is one among them. Earlier we enabled few Cypher suits (SHA512, PKCS5 etc) in similar way.

Reply:
I found and understood. Thank you for explanation.